src/sentry/utils/pytest/selenium.py

```diff
@@ -224,9 +224,10 @@ def _environment(request):
     config._environment.append(('Driver', config.option.selenium_driver))
 
 
-@pytest.mark.tryfirst
-def pytest_runtest_makereport(item, call, __multicall__):
-    report = __multicall__.execute()
+@pytest.hookimpl(hookwrapper=True)
```

Comment:
tbh waiting on Travis for this one to confirm it actually worked. I hobbled this together from random PRs that fixed this warning themselves by doing something similar.